eepget.cpp

```diff
@@ -1,28 +1,36 @@
-// Copyright (c) 2017 The I2P Project
-// Distributed under the MIT software license, see the accompanying
-// file LICENSE or http://www.opensource.org/licenses/mit-license.php.
-//--------------------------------------------------------------------------------------------------
+/**
+ * Copyright (c) 2017 The I2P Project
+ *
+ * Distributed under the MIT software license, see the accompanying
+ * file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+ *
+ * See full documentation about SAM at http://www.i2p2.i2p/samv3.html
+ */
 
 #include "i2psam.h"
 
-#include <iostream>
-
 int main(int argc, char **argv)
 {
-    if (argc < 2) {
-        std::cerr << "Usage: eepget <hostname.i2p>" << std::endl;
-        return 1;
+  if (argc < 2)
+    {
+      std::cerr << "Usage: eepget <hostname.i2p>" << std::endl;
+      return 1;
     }
-    std::string target(argv[1]);
-    SAM::StreamSession s("eepget");
-    auto lookupResult = s.namingLookup(target);
-    auto connResult = s.connect(lookupResult.value, false);
-    auto conn = connResult.value.get();
-    conn->write("GET / HTTP/1.1\r\n\r\n");
-    auto reply = conn->read();
-    while (!reply.empty()) {
-        std::cout << reply << std::flush;
-        reply = conn->read();
+
+  std::string target(argv[1]);
+  SAM::StreamSession s("eepget");
+  auto lookupResult = s.namingLookup(target);
+  auto connResult = s.connect(lookupResult.value, false);
+  auto conn = connResult.value.get();
+  conn->write("GET / HTTP/1.1\r\n\r\n");
+  auto reply = conn->read();
+
+  while (!reply.empty())
+    {
+      std::cout << reply << std::flush;
+      reply = conn->read();
     }
-    conn->close();
+
+  std::cout << std::endl;
```

Comment:
I'm just trying to follow [POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) standards.

Output befor:
```
[user@pc]$ ./eepget stats.i2p
...
</body></html>[user@pc]$

```
And after:
```
$ ./eepget stats.i2p
...
</body></html>
[user@pc]$
```

We can remove it, it's just a decorative change.

Reply:
I think probably better to remove it. It's probably not a big problem because this eepget is mostly just a demo, but supposing a situation where somebody was using eepget to download a file which had been hashed and signed, and which did not end with a newline character. In some cases, when using this eepget, the newline character may cause the hash to change and the signature to be invalidated. Just write out the file as-is for now please.

Reply:
[Done](https://github.com/i2p/i2psam/pull/10/commits/1d9577a09b7fe45fe416187c5e10e3e96c6093b7)